app/assets/javascripts/select-kit/addon/components/period-chooser/period-chooser-header.hbs

```diff
@@ -1,4 +1,4 @@
-<h2 class="selected-name" title={{this.title}} role="option">
+<h2 class="selected-name" title={{this.title}}>
```

Comment:
`option` is not a valid role in all three of these places - they're not selectable options (they select-kit headers) and don't have aria-selected attr